.github/workflows/ci.yml

```diff
@@ -139,6 +139,50 @@ jobs:
           github-token: ${{ secrets.github_token }}
           parallel-finished: true
 
+  docker:
+    needs:
+      - lint
+      - test-unit
+      - test-integration
+      - test-integration-windows
+      - validate-components
+    # Only run on tags starting with v prefix for now -- extra push need for triggering CI again
+    # if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            solidproject/community-server
+          tags: |
+            type=raw,value=2.0.1
+            type=raw,value=2.0
+            type=raw,value=2
+            type=raw,value=latest
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+          github-token: ${{ secrets.github_token }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
@joachimvh Do you have any idea why these variables are not being picked up? @RubenVerborgh Added them as repository secrets yesterday, and I should have write access. So normally my PR from my forked repo should be able to use these secrets in the triggered workflow/action.

Reply:
So I found this https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/ (second section on public repos)

I think that if we change [this line](https://github.com/Falx/community-server/blob/feat/docker-image/.github/workflows/ci.yml#L10) to `pull_request_target:`

That the proper write access is granted to PR pipeline runs (since the owner of the workflow will be the original repo and not the fork). This would however give everyone forking and PR-ing access to the secrets. I can't decide if that is ok..

Reply:
Indeed, just create a new PR with a branch from this repo.

Reply:
Ok, I will close this PR and do a new one from a branch on this repo. That should work.